lib/package.json

```diff
@@ -68,8 +68,8 @@
     },
     "peerDependencies": {
         "@babel/runtime-corejs3": "^7.11.2",
-        "react": "^16.14.0",
-        "react-dom": "16.14.0",
+        "react": "^16.14.0 || ^17.0.0",
```

Comment:
Shall we go with. WDYT?

```
"react": "16.14.x || 17.x",
```

Reply:
IMO , Since this is `react` it won't be a big issue, But using [x-ranges](https://docs.npmjs.com/cli/v6/using-npm/semver#x-ranges-12x-1x-12-) is flexible(open to version bumps) than the [Caret Ranges](https://docs.npmjs.com/cli/v6/using-npm/semver#caret-ranges-123-025-004).

Reply:
I think the `^` does the same, right?

Reply:
I think better to go with ```>=16.14 || 17.x```. Since my original suggestion will not allow `16.15.x`.

Reply:
Then we could use just "react": ">=16.4" , Is 16.4 because of hooks ?

Reply:
+
I think we might have to go with `>=16.8` then.

Reply:
Done update the version rage as above `>=16.8`